src/gui/properties/speedplotview.h

```diff
@@ -110,20 +111,17 @@ class SpeedPlotView : public QGraphicsView
         bool enable;
     };
 
+    int maxYValue();
+    boost::circular_buffer<PointData> &getCurrentData();
+
     boost::circular_buffer<PointData> m_data5Min;
     boost::circular_buffer<PointData> m_data30Min;
     boost::circular_buffer<PointData> m_data6Hour;
     QMap<GraphID, GraphProperties> m_properties;
-
```

Comment:
is grouping them together a good idea?

Reply:
When you fix coding style issues you shouldn't do changes fixing no such issues.

Reply:
ok.